assets/js/setup.tsx

```diff
@@ -60,4 +61,21 @@ export const loadSimpleJekyllSearch = () => {
   });
 };
 
+export const setUpTimezoneHooks = () => {
+  Array.from(
+    document.querySelectorAll<HTMLElement>("[data-local-tzname]")
+  ).forEach((elem) => {
+    elem.innerText = localTimezoneName();
+  });
```

Comment:
You could use [`textContent`](https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent) to avoid specifying the generic. (Same applies below)

```suggestion
  Array.from(document.querySelectorAll("[data-local-tzname]")).forEach(
    (elem) => {
      elem.textContent = localTimezoneName();
    }
  );
```

Reply:
went with customelements approach in a46c3555e65a46619e7d56b858b1ed87dbfac7da